windows-release/azure-pipelines.yml

```diff
@@ -218,7 +224,7 @@ stages:
     - ${{ if eq(parameters.DoMSI, 'true') }}:
       - stage: PublishPyDotOrg
         displayName: Publish to python.org
-        dependsOn: ['Test_MSI', 'Test']
+        dependsOn: ['SBOM', 'Test_MSI', 'Test']
         jobs:
         - template: stage-publish-pythonorg.yml
```

Comment:
I assume we're going to get chances to this template as well to SSH the files up to the server?

Reply:
Yeah that's the plan, I might do that in a follow-up PR though.